packages/hash/frontend/src/shared/edit-bar-scroller.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from "@mui/material";
+import { createContext, ReactNode, useContext, useMemo, useState } from "react";
+
+const EditBarContext = createContext<{
+  page: HTMLElement;
+  scrollingNode: HTMLElement;
+} | null>(null);
+
+export const useEditBarContext = () => useContext(EditBarContext);
```

Comment:
We can potentially add _if `null`, `throw`_ here. This will allow us to do `const { page, scroller } = useEditBar()`. We only use this hook in one place though so we can probably leave the signature as is and revisit it if we see that this hook has become more popular.

Reply:
I thought about that but would mean you would need to check both variables are defined in the effect. Will leave this as is for now. 

Thank you!